2021-09-30 20:01:08+0100 [-] Log opened.
2021-09-30 20:01:08+0100 [-] /home/lnagy2/VirtualEnvs/m4db/lib/python3.9/site-packages/urllib3/connectionpool.py:1013: urllib3.exceptions.InsecureRequestWarning: Unverified HTTPS request is being made to host 'localhost'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
